README.md

```diff
@@ -1 +1,17 @@
-# github-status
\ No newline at end of file
+# github-status
+
+## Deployment
+Build Docker container with:
+
+`docker build -f ./Dockerfile -t githubstatus .`
+
+Then run it like this:
+
+`docker run --env BOT_TOKEN=<token> --env CHAT_ID=<chat_id> githubstatus`
+
+## Run flake8
+`flake8 --extend-ignore E501 --exclude venv/`
+
+## Run tests:
+`pip install -r test-requirements.txt`\
+`python -m pytest -v test/`
```

Comment:
It looks strange, why is CHAT_ID required from me when testing locally?

```
$ pip install -r test-requirements.txt
...
$ python -m pytest -v test/
============================= test session starts ==============================
platform linux -- Python 3.9.1, pytest-7.3.1, pluggy-1.2.0 -- /home/pc/workspace/vk/github-status/venv/bin/python
cachedir: .pytest_cache
rootdir: /home/pc/workspace/vk/github-status
plugins: requests-mock-1.10.0
collected 5 items / 2 errors                                                   

==================================== ERRORS ====================================
...
=========================== short test summary info ============================
ERROR test/test_client.py - AssertionError: Environment variable CHAT_ID not set.
ERROR test/test_vkteamsbot.py - AssertionError: Environment variable CHAT_ID not set.
!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!
============================== 2 errors in 0.10s ===============================
```

Reply:
Fixed